examples/pxScene2d/src/Makefile

```diff
@@ -142,6 +142,10 @@ SEARCH=\
 #CXXFLAGS=-Wall -Wextra -g $(SEARCH) -DRT_PLATFORM_LINUX -D$(PX_PLATFORM) -DENABLE_RT_NODE
```

Comment:
I don't believe any Makefile changes are needed at this time.  Let's omit all Makefile changes.